sdk/agrifood/agrifood-farming-rest/test/public/farmHeirarchy.spec.ts

```diff
@@ -111,10 +111,11 @@ describe("Farmer Operations", () => {
     const poller = getLongRunningPoller(client, initialResponse);
     const result = await poller.pollUntilDone();
 
-    assert.equal(result.body.boundaryId, boundaryId);
+    const jobOutput = <SatelliteDataIngestionJobOutput>result.body;
```

Comment:
Same as other comments, if you check and throw with isUnexpected you don't need to cast